XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/BaseController.cs

```diff
@@ -109,7 +109,7 @@ public bool SetupConfiguration(Type controllerType)
             {
                 // Assign any known interaction mappings.
                 if (controllerMappings[i].ControllerType?.Type == controllerType &&
-                    controllerMappings[i].Handedness == ControllerHandedness &&
+                    (controllerMappings[i].Handedness == ControllerHandedness || controllerMappings[i].Handedness == Handedness.Both) &&
```

Comment:
Still wondering how this will impact mappings

Reply:
I've tested all the major platforms without issue.  And so far the only "mappings" we actively support are Left, Right and now Both.